config/karma.dev.conf.js

```diff
@@ -25,10 +25,9 @@ const files = base.deps
   .concat(base.units)
   .concat(base.integrations);
 const wp = ['webpack'];
-const preprocessors = base.helpers.reduce((acc, key) => {
-  acc[key] = wp;
-  return acc;
-}, {});
+const preprocessors = {
```

Comment:
It looks like this is the same for all configurations, yes? We can bump it up to `karma.conf.js`.